src/core/trulens/core/database/orm.py

```diff
@@ -315,6 +319,68 @@ def parse(
                     multi_result=obj.multi_result,
                 )
 
+        class GroundTruth(base):
+            """
+            ORM class for [GroundTruth][trulens.core.schema.groundtruth.GroundTruth].
+
+            Warning:
+                We don't use any of the typical ORM features and this class is only
+                used as a schema to interact with database through SQLAlchemy.
+            """
+
+            _table_base_name = "ground_truth"
+
+            ground_truth_id = Column(TYPE_ID, nullable=False, primary_key=True)
+            dataset_id = Column(Text, nullable=False)
+            ground_truth_json = Column(TYPE_JSON, nullable=False)
+
+            dataset = relationship(
+                "Dataset",
+                backref=backref("ground_truths", cascade="all,delete"),
+                primaryjoin="Dataset.dataset_id == GroundTruth.dataset_id",
+                foreign_keys=dataset_id,
+                order_by="(GroundTruth.ground_truth_id)",
+            )
+
+            @classmethod
+            def parse(
+                cls,
+                obj: mod_groundtruth_schema.GroundTruth,
+                redact_keys: bool = False,
+            ) -> ORM.GroundTruth:
+                return cls(
+                    ground_truth_id=obj.ground_truth_id,
+                    ground_truth_json=obj.model_dump_json(
+                        redact_keys=redact_keys
+                    ),
+                    dataset_id=obj.dataset_id,
+                )
+
+        class Dataset(base):
+            """
+            ORM class for [Dataset][trulens.core.schema.dataset.Dataset].
+
+            Warning:
+                We don't use any of the typical ORM features and this class is only
+                used as a schema to interact with database through SQLAlchemy.
+            """
+
+            _table_base_name = "dataset"
+
+            dataset_id = Column(TYPE_ID, nullable=False, primary_key=True)
+            dataset_json = Column(TYPE_JSON, nullable=False)
```

Comment:
You intend to be loading the whole "dataset" at once? How big will these be?

Reply:
Ohh, nevermind. Didn't realize this is just the meta info with GroundTruth being the actual "rows".

Reply:
yes. Dataset is only holding properties / metadata of the dataset, not the data themselves.